### PR TITLE
Updated Service accounts

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated enhanced Service accounts add-on pricing.

### CODE REVIEW

File: docker-hub/service-accounts.md [hunk 1]
```diff
@@ -16,10 +16,11 @@ Refer to the following table for details on the Enhanced Service Account add-on
...
 | ------ | ------ | ------ |
-| 1 | 15-50k | $9,950/yr |
-| 2 | 50-150k | $17,950/yr |
-| 3 | 150k-500k | $60,000/yr |
-| 4 | 500k+ | Tier 4+ $60k/yr/500k Pull increment |
+| 1 | 5-10k | $9,950/yr |
+| 2 | 10-25k | $17,950/yr |
+| 3 | 25k-50k | $32,950/yr |
+| 4 | 50-100k+ | $58,950/yr |
+| 5 | 100k+ | Contact Sales |
```
Comment (usha-mandya) [inline docker-hub/service-accounts.md:23]: @jeremy-castile Should we add a link here for the Contact Sales option

File: docker-hub/service-accounts.md [hunk 2]
```diff
@@ -16,10 +16,11 @@ Refer to the following table for details on the Enhanced Service Account add-on
...
 | Tier | Pull Rates Per Day* | Annual Fee |
 | ------ | ------ | ------ |
-| 1 | 15-50k | $9,950/yr |
-| 2 | 50-150k | $17,950/yr |
-| 3 | 150k-500k | $60,000/yr |
-| 4 | 500k+ | Tier 4+ $60k/yr/500k Pull increment |
+| 1 | 5-10k | $9,950/yr |
+| 2 | 10-25k | $17,950/yr |
+| 3 | 25k-50k | $32,950/yr |
+| 4 | 50-100k+ | $58,950/yr |
```
Comment (jeremy-castile) [inline docker-hub/service-accounts.md:22]: 50-100k ... remove the '+'

File: docker-hub/service-accounts.md (hunk 1 above)
Comment (jeremy-castile) [reply docker-hub/service-accounts.md:23]: oh. yes, good catch. Let's link to this form: https://www.docker.com/pricing/questions

**Review by usha-mandya — APPROVED** [review]
LGTM

File: docker-hub/service-accounts.md [hunk 3]
```diff
@@ -16,10 +16,11 @@ Refer to the following table for details on the Enhanced Service Account add-on
...
 | ------ | ------ | ------ |
-| 1 | 15-50k | $9,950/yr |
-| 2 | 50-150k | $17,950/yr |
-| 3 | 150k-500k | $60,000/yr |
-| 4 | 500k+ | Tier 4+ $60k/yr/500k Pull increment |
+| 1 | 5-10k | $9,950/yr |
+| 2 | 10-25k | $17,950/yr |
+| 3 | 25k-50k | $32,950/yr |
+| 4 | 50-100k | $58,950/yr |
+| 5 | 100k | [Contact Sales](https://www.docker.com/pricing/questions){:target="_blank" rel="noopener" class="_"} |
```
Comment (jeremy-castile) [inline docker-hub/service-accounts.md:23]: Sorry. This one does get the "+" sign... so "100k+"

Comment (usha-mandya) [reply docker-hub/service-accounts.md:23]: This is fixed in https://github.com/docker/docker.github.io/pull/13004 as the existing PR was merged.

**Pull request merged**